scripts/draft-translation/translate.py

```diff
@@ -1,71 +1,93 @@
 #!/bin/bash python3
 
-from googletrans import Translator  # https://github.com/ssut/py-googletrans
+import click
 
-BASE_PATH = "../.."
-SOURCE_LANG="en"
-TARGET_LANG="ko"
-SOURCE_DOC=f"{BASE_PATH}/content/en/docs/tasks/debug/debug-cluster/crictl.md"
-TARGET_DOG=f"{BASE_PATH}/content/ko/docs/tasks/debug/debug-cluster/crictl.draft.md"
-BASE_DIR="../../"
+from googletrans import Translator  # https://github.com/ssut/py-googletrans
 
 SKIP_PREFIXES = [
-    "---",
     "<!--",
-    "id:",
-    "title:",
-    "date:",
-    "full_link:",
-    "short_description:",
-    "aka:",
-    "tags:",
-    "reviewers:",
-    "approvers:",
-    "content_type:",
-    "#",
-    "{{",
+    "## {{% heading",
+    "{{< feature-state",
+    "{{< comment",
+    "{{< /comment",
+    "{{< glossary_definition",
+    "{{< note",
+    "{{< /note",
+    "{{< caution",
+    "{{< /caution",
+    "{{< table",
+    "{{< /table",
+    "{{% thirdparty",
+    "{{< codenew",
 ]
 
-if __name__ == '__main__':
+@click.command()
+@click.option('--lang', '-t', required=True, help='Short code (ex:ko) for Target language for draft localization')
+@click.option('--file', '-f', required=True, help='File path of the Source document')
+@click.option("--src-lang", '-s', required=False, help='Short code (ex:en) for Source language', default="en")
+def main(lang, file, src_lang):
+    """Simple program that create a draft file for localization using machine tranlation."""
+
+    target_lang = lang
```

Comment:
`lang` 변수가 `target_lang`으로 치환되고 뒤에 사용되지 않는 거면 요렇게 바꾸는게 좋을 것 같습니다.
```suggestion
def main(target_lang, file, src_lang):
    """Simple program that create a draft file for localization using machine tranlation."""
```